<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddSerialsRangeDialog</class>
 <widget class="QDialog" name="AddSerialsRangeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>194</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add serials range</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelFrom">
       <property name="text">
        <string>From:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelTo">
       <property name="text">
        <string>To:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelCount">
       <property name="text">
        <string>Count:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="labelCountDisplay"/>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="spinBoxFrom">
       <property name="maximum">
        <number>2147483647</number>
       </property>
       <property name="value">
        <number>999930</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinBoxTo">
       <property name="maximum">
        <number>2147483647</number>
       </property>
       <property name="value">
        <number>999940</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddSerialsRangeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddSerialsRangeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
